website/templates/util/render_keys.mako

```diff
@@ -1,3 +1,4 @@
+<!-- TODO: File is unused; candidate for deletion? -->
```

Comment:
Probably so. This was probably what we used for developer keys back in the v1 days. I say "used" but I don't think it saw much use outside of an early version of SHARE and maybe a couple of edge cases.